Sources_v3/StreamChatUI/ChatChannelList/ChatChannelList.swift

```diff
@@ -0,0 +1,108 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class ChatChannelList<ExtraData: ExtraDataTypes>: UICollectionViewController
```

Comment:
`ChatChannelListVC` `ChatChannelListController` ??

Reply:
renamed in 47b487dba9229c15cc2c2858b828f8e6cde80c65